dex/networks/btc/script.go

```diff
@@ -494,6 +557,18 @@ func RefundP2SHContract(contract, sig, pubkey []byte) ([]byte, error) {
 		Script()
 }
 
+// RefundP2WSHContract returns the witness to refund a contract output
+// using the contract author's signature after the locktime has been reached.
+// This function assumes P2WSH and appends the contract as the final data push.
+func RefundP2WSHContract(contract, sig, pubkey []byte) [][]byte {
+	return [][]byte{
+		sig,
+		pubkey,
+		[]byte{},
```

Comment:
`[]byte{0}`? Or does this turn into a 0 anyway when the TxIn/MsgTx is processed?

Reply:
Yes, it does. It took me a while to figure this out, but it has to be minimally encoded by policy. https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2016-August/013014.html